src/ConnectedMode/UI/ManageBinding/ManageBindingDialog.xaml.cs

```diff
@@ -66,6 +69,10 @@ private void SelectProject_OnClick(object sender, RoutedEventArgs e)
     private async void ManageBindingDialog_OnInitialized(object sender, EventArgs e)
     {
         await ViewModel.InitializeDataAsync();
+        if (useSharedBindingOnInitialization)
+        {
+            await ViewModel.UseSharedBindingWithProgressAsync();
```

Comment:
After showing the `Connection not found` warning the Manage binding window opens in a weird state (With the message that binding cannot be established, while this is kinda true, it's also a bit misleading because there is nothing on the Output logs that will help the user).

![image](https://github.com/user-attachments/assets/d218345d-6b42-4280-8459-1f90defcb03d)

Something might not be cleaned properly 🤔 .

Reply:
Can you give more details when is this happening. Not sure when is "After showing the Connection not found warning"

Reply:
You mean when the connection is not in the connections.json?

Reply:
Sorry. It passed a day since I worked on this. I got the context. :P

Reply:
Yes, when the connection does not exist, but a shared binding expects it.
The user clicks "bind" on the gold bar, and the warning message box appears. After closing the message box, the manage binding window opens with the above error.